unlocode-mdb-to-csv.sh

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
```

Comment:
Probably commits in this PR should be rebased into 2 commits. One about new source of data and another one about version changes.

Reply:
Also, add notes about it to the change log.

Reply:
As planned, upgrade to the new version will be in a separate pr